samples/demo-component/demo-app/archetype/config/webpack/webpack.config.dev.js

```diff
@@ -5,7 +5,8 @@ const repoPackagesDir = Path.join(__dirname, "../../../../packages");
 module.exports = {
   resolve: {
     alias: {
-      "demo-component": Path.join(repoPackagesDir, "demo-component/src")
+      "demo-component": Path.join(repoPackagesDir, "demo-component/src"),
+      "react": Path.resolve(__dirname, "../../../node_modules", "react")
```

Comment:
This line is required due to `specifically state where to use the required version of react`

Reference: https://github.com/JedWatson/react-select/issues/2025